infra/modules/providers/azure/service-plan/variables.tf

```diff
@@ -0,0 +1,27 @@
+variable "resource_group_name" {
+  description = "Resource group name that the service plan will be created in."
+  default     = "cblt-svcplan-rg"
```

Comment:
the naming convention for a resource group should be more general ie remove `svcplan`